core/src/main/java/org/kohsuke/stapler/Function.java

```diff
@@ -226,7 +226,7 @@ Object bindAndInvoke(Object o, StaplerRequest req, StaplerResponse rsp, Object..
             throw new AssertionError(e);    // impossible
         }
 
-        PARSE_METHODS = CacheBuilder.newBuilder().weakKeys().build(new CacheLoader<Class,Function>() {
+        PARSE_METHODS = Caffeine.newBuilder().weakKeys().build(new CacheLoader<Class,Function>() {
```

Comment:
I doubt this ever worked to begin with. See https://github.com/jglick/biweak-class-map

Reply:
> I doubt this ever worked to begin with

This would not have worked in the case of a class with a `fromStapler` method, because then the value would be a `MethodFunction` whose `Method m` has a `Class<?> clazz` that refers to the key of the map.

But it would have worked in the case of a class _without_ a `fromStapler` method, because then the value would be a `StaticFunction` whose `Method m` has a `Class<?> clazz` that refers to `org.kohsuke.stapler.Function` rather than the key of the map.